src/main/java/com/joyent/manta/client/MantaClient.java

```diff
@@ -338,81 +290,221 @@ public void deleteRecursive(final String path) throws MantaCryptoException, Mant
             }
         }
 
-        LOG.debug(String.format("finished deleting path %s", path));
+        LOG.debug("Finished deleting path {}", path);
     }
 
 
     /**
-     * Get a Manta object.
+     * Get the metadata for a Manta object. The difference with this method vs head() is
+     * that the request being made against the Manta API is done via a GET.
      *
      * @param path The fully qualified path of the object. i.e. /user/stor/foo/bar/baz
-     * @return The {@link MantaObject}.
-     * @throws IOException If an IO exception has occured.
-     * @throws MantaCryptoException If there's an exception while signing the request.
+     * @return The {@link MantaObjectMetadata}.
+     * @throws IOException If an IO exception has occurred.
+     * @throws com.joyent.manta.exception.MantaCryptoException If there's an exception while signing the request.
      * @throws MantaClientHttpResponseException If a http status code {@literal > 300} is returned.
      */
-    public MantaObject get(final String path) throws MantaCryptoException, MantaClientHttpResponseException,
-    IOException {
-        LOG.debug(String.format("entering get with path %s", path));
+    public MantaObjectMetadata get(final String path) throws IOException {
+        final HttpResponse response = httpGet(path);
```

Comment:
Does this `HttpResponse` instance need to be explicitly disconnected?  I noticed the `httpPut` method contains a finally block which explicitly invokes the disconnect method of the response, but I don't see anything explicit here or in any of the invokers of this method.  Could this be causing the leak that I'm seeing with the http client's connection pool?